products/compute/terraform.yaml

```diff
@@ -1047,6 +1047,10 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         input: false
         default_from_api: true
         custom_flatten: 'templates/terraform/custom_flatten/name_from_self_link.erb'
+      enableFlowLogs: !ruby/object:Overrides::Terraform::PropertyOverride
+        default_from_api: true
```

Comment:
This means that if I remove `enable_flow_logs = true` from my config, the value will actually remain set. Do we expect the value to be set by the API when it wasn't before? I'm fairly hesitant to make this change.

Reply:
Removed